interview-questions.md

```diff
@@ -8,44 +8,92 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What does calling super do in a Ruby class?
 
-Your answer:
+Your answer: Calling a super is like using a youtube minecraft tutorial(class) to build a house /mansion as a starting point for how you want it to look like but modifying based on what you want. 
+
+
+Researched answer: "When a method with arguments is overridden by one of its child classes then a call to super without any argument in the child method will automatically pass the arguments of the child method to the parent method."
+
+
+
+
 
-Researched answer:
 
 2. What is a gem?
 
-Your answer:
+Your answer: In ruby its a packaged library or application that is used to manage & distribute your projects in ruby.
+
+Researched answer: "A gem is a bunch of perfectly packaged Ruby code that gives you access to custom methods somebody else wrote. Gems just make the process of sharing code much easier. The custom methods inside of gems typically have specific functionalities."
```

Comment:
Your answers to the researched answer portion of the interview questions should be in your own words, not copied and pasted from your research. Please resubmit these answers in your own words.